CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## [Unreleased]
 ### Added
-- Technical - On start, create a forestadmin-schema.json file that contains the apimap definition.
+- Developer Experience - On start, create a .forestadmin-schema.json file that contains the schema definition.
+- Developer Experience - On production, load .forestadmin-schema.json for schema update.
+- Developer Experience - Developers can desactivate the automatic schema synchronisation on server start using the `FOREST_DISABLE_AUTO_SCHEMA_APPLY` environment variable.
```

Comment:
`OnlyForRelationships` collections are displayed in my navigation panel, to fix I guess.